source/user-manual/api/configuration.rst

```diff
@@ -324,6 +327,20 @@ limits
 | allow      | yes, true, no, false | true          | Allow uploading configurations with modified EPS limits through the Wazuh API. Setting this option to ``false`` prevents uploading ``ossec.conf`` files if the ``<limits><eps>`` section inside the :ref:`global tag <reference_ossec_global>` has changed.                                                                                 |
 +------------+----------------------+---------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+agents
+~~~~~~
+
+.. rubric:: allow_higher_versions
+   :class: h5
+
+.. versionadded:: 4.6.0
+
++------------+----------------------+---------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Sub-fields | Allowed values       | Default value | Description                                                                                                                                                                                                                                                                                                                                 |
++============+======================+===============+=============================================================================================================================================================================================================================================================================================================================================+
+| allow      | yes, true, no, false | true          | Allow uploading configurations that accept higher agent versions through the Wazuh API. Setting this option to ``false`` prevents uploading ``ossec.conf`` files that contain the ``<agents><allow_higher_versions>`` section with the ``yes`` value inside the :ref:`global tag <reference_ossec_global>`.                                 |
```

Comment:
After the last changes made [here](https://github.com/wazuh/wazuh/pull/18342), this part needs to be updated:

```suggestion
| allow      | yes, true, no, false | true          | Allow uploading configurations that accept higher agent versions through the Wazuh API. Setting this option to ``false`` prevents uploading ``ossec.conf`` files that contain the ``<allow_higher_versions>`` section with the ``yes`` value inside the :ref:`auth <reference_ossec_auth>` or :ref:`remote <reference_ossec_remote>` tags.  |
```